src/test/kotlin/io/github/mojira/arisa/modules/HelpersTest.kt

```diff
@@ -564,33 +450,25 @@ class HelpersTest : StringSpec({
             "https://bugs.mojang.com/browse/MC-312", "https://bugs.mojang.com/browse/MC-400"
         )
         list.convertLinks()
-        list shouldBe (mutableListOf("MC-1", "MC-20", "MC-312", "MC-400"))
+        list shouldBe mutableListOf("MC-1", "MC-20", "MC-312", "MC-400")
     }
 
     "convertLinks should not touch ticket numbers" {
         val list1 = mutableListOf("MC-1", "MC-20", "MC-312", "MC-400")
         list1.convertLinks()
-        list1 shouldBe (mutableListOf("MC-1", "MC-20", "MC-312", "MC-400"))
+        list1 shouldBe mutableListOf("MC-1", "MC-20", "MC-312", "MC-400")
 
         val list2 = mutableListOf(
             "https://bugs.mojang.com/browse/MC-1", "MC-20",
             "https://bugs.mojang.com/browse/MC-312", "MC-400"
         )
         list2.convertLinks()
-        list2 shouldBe (mutableListOf("MC-1", "MC-20", "MC-312", "MC-400"))
+        list2 shouldBe mutableListOf("MC-1", "MC-20", "MC-312", "MC-400")
     }
 
     "in created LinkType nameVariants should contain a sorted set of all possible combinations (without skipping middle item and without changing order) of concatenated elements of name list" {
-        LinkType(listOf("1", "2", "3"), "123", true).nameVariants shouldBe (
-            sortedSetOf(
-                "1",
-                "1 2",
-                "1 2 3",
-                "2",
-                "2 3",
-                "3"
-            )
-            )
+        LinkType(listOf("1", "2", "3"), "123", true).nameVariants
+            .shouldContainExactly("1", "1 2", "1 2 3", "2", "2 3", "3")
```

Comment:
Changed this to `shouldContainExactly` because that makes sure the collection contains these elements in the specified order. `shouldBe sortedSetOf(...)` apparently would have performed a regular equality check ignoring order. For example `setOf("b", "a") shouldBe sortedSetOf("a", "b")` would pass.